### PR TITLE
Add WT32-SC01 PLUS support

### PR DESCRIPTION
**Pull request by c4801725870**
-----------
## Description of Change
Added pins_arduino.h and a boilerplate boards.txt entry for the Panlee WT32-SC01 Plus.

## Tests scenarios
Test on my WT32-SC01 Plus. 

## Related links
Please provide links to related issue, PRs etc.
https://www.marutsu.co.jp/contents/shop/marutsu/datasheet/khe_WT32-SC01-PLUS.pdf

## Summary
Added pins_arduino.h and a boilerplate boards.txt entry for the Panlee WT32-SC01 Plus.

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -22771,6 +22771,158 @@ wt32-eth01.menu.EraseFlash.all.upload.erase_cmd=-e
...
+wt32-sc01-plus.menu.PSRAM.disabled=Disabled
+wt32-sc01-plus.menu.PSRAM.disabled.build.defines=
+
+wt32-sc01-plus.menu.PartitionScheme.default_8MB=Default (3MB APP/1.5MB SPIFFS)
+wt32-sc01-plus.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+wt32-sc01-plus.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2=TinyUF2 Compatibility (2MB APP/3.7MB FFAT)
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.build.custom_bootloader=bootloader_tinyuf2
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.build.custom_partitions=partitions_tinyuf2
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.upload.extra_flags=0x410000 "{runtime.platform.path}/variants/{build.variant}/tinyuf2.bin"
```
Comment (P-R-O-C-H-Y) [inline boards.txt:22868]: You are missing the `tinyuf2.bin` file in the variant folder.

File: variants/wt32-sc01-plus/pins_arduino.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+static const uint8_t SD_CLK = 39;
+static const uint8_t SS    = 41;
+static const uint8_t MOSI  = 40;
+static const uint8_t MISO  = 38;
+static const uint8_t SCK   = 39;
+
+
+
+
+
```
Comment (P-R-O-C-H-Y) [inline variants/wt32-sc01-plus/pins_arduino.h:68]: Remove empty lines
```suggestion
```

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@c4801725870 PTAL

File: boards.txt (hunk 1 above)
Comment (c4801725870) [reply boards.txt:22868]: > You are missing the `tinyuf2.bin` file in the variant folder.

Sorry I updated it. Also I am not sure how to squash commits, sorry about that.

**Review by P-R-O-C-H-Y — APPROVED** [review]

**Pull request merged**